src/game/overlay.c

```diff
@@ -213,7 +206,14 @@ static void Overlay_BarDraw(BAR_INFO *bar_info)
 
     int32_t x = 0;
     int32_t y = 0;
-    Overlay_BarGetLocation(bar_info->location, width, height, &x, &y);
+    if (bar_info->location == BL_CUSTOM) {
+        width = bar_info->custom_width;
+        height = bar_info->custom_height;
+        x = bar_info->custom_x;
+        y = bar_info->custom_y;
+    } else {
+        Overlay_BarGetLocation(bar_info->location, width, height, &x, &y);
+    }
```

Comment:
Can we change `Overlay_BarGetLocation` to also make width and height writable, and move the `BL_CUSTOM` case inside?

Reply:
LMK if the change I made for this is OK. Since I pass `BAR_INFO` now.